src/cf-tables/src/cf-tables.less

```diff
@@ -81,20 +97,24 @@
 }
 
 .respond-to-min( @bp-sm-min, {
-    .table__striped {
+    .o-table__striped {
         .striped-table();
     }
-});
+} );
 
 .respond-to-max( @bp-xs-max, {
-    .table__striped > tbody > tr:nth-child(even) {
+    .o-table {
+        width: 100%;
+    }
+
+    .o-table__striped > tbody > tr:nth-child(even) {
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/396/commits/5ec3a501b3faa2a5060d58777d15bcf79568b265